src/can/can.c

```diff
@@ -57,28 +99,26 @@ uint8_t load_data(mob_t* mob) {
     return len;
 }
 
+// Tests to see if the selected mob is paused
 uint8_t is_paused(mob_t* mob) {
     select_mob(mob->mob_num);
 
-    // the MOb is paused iff the upper two bits of CANCDMOB are 0
+    // the MOb is paused iff the two MSB of CANCDMOB are 0
     if (CANCDMOB & 0xc0) {
         return 0;
     } else {
         return 1;
     }
 }
 
-void init_can() {
+// Initializes CAN protocol
+void init_can(void) {
+    // Reset CAN controller and set communication baud rate to default (100 kbps)
+    void set_can_baud_rate(can_baud_rate_t baud_rate); // Need to formally declare this function first.
```

Comment:
Don't declare a function within another function - if you're not planning to expose it in the header file, put it at the top of this file.